_posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "My Introduction to Wi-Fi 'Hard block' and 'rfkill'"
+date: 2020-07-26T15:34:30-04:00
```

Comment:
You have written in the future!

Please set the date correct. Its 26th July as per the timestamp

Reply:
Changed it, thanks for pointing out !